### PR TITLE
Fix groups in ILS reports

### PR DESCRIPTION
**Pull request by lwyszomi**

### CODE REVIEW

File: custom/ilsgateway/api.py [hunk 1]
```diff
@@ -236,7 +236,7 @@ def location_sync(self, ilsgateway_location, fetch_groups=False):
             location.domain = self.domain
             location.name = ilsgateway_location.name
             if ilsgateway_location.groups:
-                location.metadata = {'groups': ilsgateway_location.groups}
+                location.metadata = {'groups': ilsgateway_location.groups[0]}
```
Comment (czue) [inline custom/ilsgateway/api.py]: confirming we think this should only contain one group now? should we change the key to be singular?

**Comment by lwyszomi** [comment]
@czue yes, that's right,

Mike said:
'''We can drop any location labelled as PILOT from the migration (which includes all of these locations) so luckily we don't have to worry about this multiple groups per location problem.'''

I will change key to singular, please don't merge

**Comment by czue** [comment]
@lwyszomi ping me when you want me to look again

**Comment by lwyszomi** [comment]
@czue ready to merge

File: custom/ilsgateway/tanzania/reminders/reports.py [hunk 2]
```diff
@@ -20,5 +20,5 @@ def _construct_status_dict(status_type, status_values, supply_points, datespan):
 
 def construct_summary(supply_point, status_type, values, cutoff):
     children = supply_point.location.children
-    children = filter(lambda sp: get_current_group() in sp.metadata.get('groups', []), children)
+    children = filter(lambda sp: get_current_group() in sp.metadata.get('groups', None), children)
```
Comment (czue) [inline custom/ilsgateway/tanzania/reminders/reports.py]: i think this one should be singular too?

**Comment by czue** [comment]
it seems like any future changes would be easier if you pulled the logic out into a `get_location_group` utility function, but I won't block on that.

**Comment by lwyszomi** [comment]
@czue updated, 

I will move this logic in other PR

**Pull request merged**